.github/workflows/mac-artifact.yml

```
@@ -12,10 +12,19 @@ on:
         type: string
 jobs:
   build:
-    runs-on: macos-11
     strategy:
       matrix:
-        ev-lib: ["", libev, libuv, libevent]
+        os: [macos-11, macos-14]
+        # ev-lib: ["", libev, libuv, libevent]
+        ev-lib: [libuv]   
+        include:
+          - os: macos-11
+            openssl-path: /usr/local/opt/openssl
```

```suggestion
        os: [macos-12, macos-14]
        # ev-lib: ["", libev, libuv, libevent]
        ev-lib: [libuv]   
        include:
          - os: macos-12
            openssl-path: /usr/local/opt/openssl
```
Support for macos-11 runners has been dropped.